server/store/store.go

```diff
@@ -55,8 +55,13 @@ type pluginStore struct {
 	Tracker            tracker.Tracker
 }
 
-func NewPluginStore(api plugin.API, logger bot.Logger, tracker tracker.Tracker) Store {
+func NewPluginStore(api plugin.API, logger bot.Logger, tracker tracker.Tracker, enableEncryption bool, encryptionKey []byte) Store {
 	basicKV := kvstore.NewPluginStore(api)
+
+	if enableEncryption {
+		basicKV = kvstore.NewEncryptedKeyStore(basicKV, encryptionKey)
```

Comment:
Encrypting everything in the kv store can make things hard to debug - locally or with the customer. I suggest we just encrypt the OAuth tokens. @esarafianou Do you have any thoughts on this?

Reply:
Yeah I just added the implementation, but we can enable it on individual stores as required. I'm assuming we won't be using it locally very much 🙃

Reply:
Yes, encrypting the oauth tokens is enough. It's not required to encrypt the calendar data.